code/game/objects/structures/barricade/handrail.dm

```diff
@@ -40,6 +41,27 @@
 		if(E.icon_path && E.obj_icon_state_path)
 			overlays += image(E.icon_path, icon_state = E.obj_icon_state_path)
 
+/obj/structure/barricade/handrail/Collided(atom/movable/movable)
+	if(istype(movable,/mob/living/carbon/xenomorph/ravager) || istype(movable,/mob/living/carbon/xenomorph/crusher))
+		var/mob/living/carbon/xenomorph/xenomorph = movable
+		if(!xenomorph.stat)
+			visible_message(SPAN_DANGER("[xenomorph] plows straight through [src]!"))
+			deconstruct(FALSE)
+	else
+		if(ismob(movable) && autoclimb)
+			var/mob/living/climber = movable
+			if(climber.a_intent == INTENT_HARM)
+				var/climbed = do_climb(climber)
+				if(climbed)
+					climber.client.move_delay += 3 DECISECONDS
+					if(prob(25))
+						if(ishuman(climber))
+							climber.apply_damage(15,BRUTE,no_limb_loss = TRUE)
```

Comment:
See https://github.com/cmss13-devs/cmss13/actions/runs/10111981060/job/27965048466?pr=6652#step:8:20

Basically either you need to type `climber` as a `/mob/living/carbon/human` or you add the missing arguments to `/mob/living/proc/apply_damage`